src/commands/trigger-pipeline.yml

```diff
@@ -1,8 +1,8 @@
 description: |
-  Triggers a pipeline on given branch. Also allows to installthe trigger script and not trigger into
+  Triggers a pipeline on given branch. Also allows to install the trigger script and not trigger into
   /tmp/swissknife/trigger_pipeline.sh
   it can be called with the arguments as follows
-  /tmp/swissknife/trigger_pipeline.sh vcstype username reponame branch '{"custom_params": "here"}'
+  /tmp/swissknife/trigger_pipeline.sh vcstype username reponame branch|tag '{"custom_params": "here"}'
```

Comment:
this wont work? since before they always just called the script with $branch it just worked. now unless they set the env var, it might go to tag mode and then calling script will break right?

Reply:
Ah, you’re right. Will adjust